bin/test_integration

```diff
@@ -14,25 +14,39 @@ Usage: ./bin/test_integation [options]
                   that needs credentials to a DB from Conjur. After running "./bin/test_integation --demo" run "oc get routes"
                   and go to "<the exposed route>/pets" in a browser. The entry "Mr. Sidecar" should be displayed on the screen.
 
-    --dap         Run the integation tests with a DAP Conjur deployment. By default the tests are run against a Conjur OSS deployment
+    --oss         Run the integration tests with a OSS Conjur deployment. By default the tests are run against a Conjur OSS deployment.
+
+    --dap         Run the integration tests with a DAP Conjur deployment. By default the tests are run against a Conjur OSS deployment.
+
+    --gke         Run the integration tests on GKE. By default the tests are run on GKE.
+
+    --oc311       Run the integration tests on Openshift v3.11. By default the tests are run on GKE.
+
+    --oc310       Run the integration tests on Openshift v3.10. By default the tests are run on GKE.
 
     -h, --help    Shows this help message.
 EOF
   exit
 }
 
 function runScriptWithSummon() {
-    summon --environment $PLATFORM -f ./summon/secrets.yml $1
+    summon --environment $SUMMON_ENV -f ./summon/secrets.yml $1
 }
 
-RUN_IN_DOCKER=false
+export RUN_IN_DOCKER=false
```

Comment:
let's not export this and export it in line 55 like we do with `CONJUR_DEPLOYMENT`. remove also the export in line 43

Reply:
good call